docker/dhs-feed/Dockerfile

```diff
@@ -0,0 +1,13 @@
+
+FROM demisto/python3-deb:3.8.6.13363
```

Comment:
Use as a base: `demisto/crypto:1.0.0.12979` which already takes care of importing crypto and is alpine based. See for example: https://github.com/demisto/dockerfiles/tree/master/docker/teams